app/views/admin/tags/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<h1><%= @title = "Tags – #{@tag}" %></h1>
+
+<h2>Taggings</h2>
+
+<%= render partial: 'scope' %>
+<%= render partial: 'filters' %>
+
+<%= link_to 'All', admin_tags_path(model_type: current_klass), class: 'btn' %>
+
+<% if @value %>
+  <%= link_to @name, admin_tag_path(tag: @name, model_type: current_klass), class: 'btn' %>
+<% end %>
+
+<% if current_klass == PublicBody %>
+  <%= link_to 'Public page', list_public_bodies_by_tag_path(@tag), class: 'btn' %>
+<% end %>
+
+<hr>
```

Comment:
let's drop this `<hr>` since it feels like it separates the filters from the list the filters operate on. The two are connected (and don't have `<hr>` elsewhere)